### PR TITLE
Source rules addition

### PR DESCRIPTION
**Pull request by ankit-privado**

### CODE REVIEW

File: rules/sources/financial_data.yaml [hunk 1]
```diff
@@ -135,7 +135,7 @@ sources:
     isSensitive: False
     sensitivity: medium
     patterns:
-      - "(?i).*(salary[^\\s/(;)#|,=!>]{0,10}(?:details|detail|slip|pay)|employee[^\\s/(;)#|,=!>]{0,10}salary|basic[^\\s/(;)#|,=!>]{0,10}salary|(?:dearness|houserent)[^\\s/(;)#|,=!>]{0,10}allowance|salary)"
+      - "(?i).*(salary[^\\s/(;)#|,=!>]{0,10}(?:details|detail|slip|pay)|employee[^\\s/(;)#|,=!>]{0,10}salary|basic[^\\s/(;)#|,=!>]{0,10}salary|(?:dearness|houserent)[^\\s/(;)#|,=!>]{0,10}allowance|salary)|(previous|prev|current|curr)[^\\s/(;)#|,=!>]*(employee|employment)[^\\s/(;)#|,=!>]*(?:pay|slip|salary)"
```
Comment (HiteshMah-Jan) [inline rules/sources/financial_data.yaml:138]: can you remove slip from here. doesn't makes sense

File: rules/sources/financial_data.yaml [hunk 2]
```diff
@@ -155,6 +155,6 @@ sources:
     isSensitive: True
     sensitivity: high
     patterns:
-      - "(?i).*(insurance[^\\s/(;)#|,=!>]*(?:policy|plan|card|claim)|(?:insurance)[^\\s/(;)#|,=!>]*(?:card|policy|claim|plan|company)|card[^\\s/(;)#|,=!>]*insurance|insurance[^\\s/(;)#|,=!>]*policy[^\\s/(;)#|,=!>]*(?:nbr|number|id|no)|insurance[^\\s/(;)#|,=!>]*claim[^\\s/(;)#|,=!>]*(?:nbr|number|id|no))"
+      - "(?i).*(insurance[^\\s/(;)#|,=!>]*(?:policy|plan|card|claim)|(?:insurance)[^\\s/(;)#|,=!>]*(?:card|policy|claim|plan|company)|card[^\\s/(;)#|,=!>]*insurance|insurance[^\\s/(;)#|,=!>]*policy[^\\s/(;)#|,=!>]*(?:nbr|number|id|no)|insurance[^\\s/(;)#|,=!>]*claim[^\\s/(;)#|,=!>]*(?:nbr|number|id|no)|ni[^\\s/(;)#|,=!>]*[^\\s/(;)#|,=!>]*(?:nbr|number|id|no))"
```
Comment (HiteshMah-Jan) [inline rules/sources/financial_data.yaml:158]: can you update to ni[-_]?(?:nbr|no|number)

File: rules/sources/personal_identification.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ sources:
     isSensitive: False
     sensitivity: low
     patterns:
-      - "(?i).*((?:first|given)[^\\s/(;)#|,=!>]{0,5}|full)name"
+      - "(?i).*((?:first|given|fore)[^\\s/(;)#|,=!>]{0,5}|full)name"
```
Comment (HiteshMah-Jan) [inline rules/sources/personal_identification.yaml:18]: `"(?i).*((?:first|given)[^\\s/(;)#|,=!>]{0,5}|fore|full)name"`

File: rules/sources/financial_data.yaml (hunk 1 above)
Comment (ankit-privado) [reply rules/sources/financial_data.yaml]: done.

File: rules/sources/financial_data.yaml (hunk 2 above)
Comment (ankit-privado) [reply rules/sources/financial_data.yaml]: done.

File: rules/sources/personal_identification.yaml (hunk 3 above)
Comment (ankit-privado) [reply rules/sources/personal_identification.yaml]: done. thank you @HiteshMah-Jan

**Review by HiteshMah-Jan — APPROVED** [review]

**Pull request merged**